### ISSUE TITLE
pyglet.libs.win32.types系统找不到指定的文件

### ISSUE THREAD

**Issue by xiezhipeng-git** [Nuitka/Nuitka#2946]
Before submitting an Issue, please review the
[Issue Guidelines](https://github.com/Nuitka/Nuitka/blob/develop/CONTRIBUTING.md#submitting-an-issue).

- Please check whether the bug was already reported or fixed.

- Please check out if the develop version of Nuitka works better for you.

  Download source, packages [from here](https://nuitka.net/doc/download.html) where you will also
  find instructions how to do it via pip or git.

If you want to post a problem/bug, to help us understand and resolve your issue please check that
you have provided at least the information below, and discard up to here:

- Nuitka version, full Python version, flavor, OS, etc. as output by *this exact* command.

  > python -m nuitka --version
2.3.9        
Commercial: None
Python: 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)]
Flavor: CPython Official
Executable: D:\my\work\test\tetris\venv\Scripts\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 11
Version C compiler: D:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.35.32215\bin\Hostx64\
x64\cl.exe (cl 14.3).
  If you are not providing the full output. The issue cannot be solved and will be considered
  `invalid`. The command outputs more than you think, and we are adding more all the time. We do not
  want to left guessing or trying things out to reproduce your issue.

- How did you install Nuitka and Python

  Did you use pip, anaconda, deb, rpm, from source, git clone, then install into a virtualenv or
  not, this is very important usually and one thing, the above command does not tell us (yet).

- The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g. `numpy`, you are
  expected to shorten this to the relevant ones.

  > python -m pip list -v
cloudpickle 3.0.0   d:\my\work\test\tetris\venv\lib\site-packages pip
colorama    0.4.6   d:\my\work\test\tetris\venv\lib\site-packages pip
gym         0.26.2  d:\my\work\test\tetris\venv\lib\site-packages pip
gym-notices 0.0.8   d:\my\work\test\tetris\venv\lib\site-packages pip
gym-tetris  2.2.2   d:\my\work\test\tetris\venv\lib\site-packages pip
nes_py      7.0.0   d:\my\work\test\tetris\venv\lib\site-packages pip
Nuitka      2.3.9   d:\my\work\test\tetris\venv\lib\site-packages pip
numpy       1.26.4  d:\my\work\test\tetris\venv\lib\site-packages pip
ordered-set 4.1.0   d:\my\work\test\tetris\venv\lib\site-packages pip
pip         24.1    d:\my\work\test\tetris\venv\lib\site-packages pip
pygame      2.5.2   d:\my\work\test\tetris\venv\lib\site-packages pip
pyglet      2.0.15  d:\my\work\test\tetris\venv\lib\site-packages pip
setuptools  65.5.0  d:\my\work\test\tetris\venv\lib\site-packages pip
tqdm        4.66.4  d:\my\work\test\tetris\venv\lib\site-packages pip
zstandard   0.22.0  d:\my\work\test\tetris\venv\lib\site-packages pip
- Many times when you get an error from Nuitka, your setup may be special

  Then even a `print("hello world")` program will not work, please try that and report that error
  instead. We do not need a report that says "Tensorflow does not work" when nothing actually works
  for you. You ought to also read the User Manual and check that your setup is actually supported
  already.

- Also supply a Short, Self Contained, Correct, Example

  That demonstrates the issue i.e a small piece of code which reproduces the issue and can be run
  with out any other (or as few as possible) external dependencies. Issues without this may will
  likely be rejected without much consideration. Often this can be as simple as importing a package,
  if this is a packaging issue, try that first.

  Pointers to repositories with usage of `pipenv` are very welcome, archives with examples are too,
  e.g. if a certain package structure is needed. This must be only source code, binaries are not
  used ever by us. But beware, that e.g. we cannot click around and stuff. Still do an effort to
  make the usage obvious. Having a compile script in the repo is perfect.

question: Nuitka gym 

  nuitka-crash-report.xml
[nuitka-crash-report.xml.txt](https://github.com/user-attachments/files/15949852/nuitka-crash-report.xml.txt)

**Comment by kayhayen** [comment]
Do you have a minimal reproducer, the crash report shows the exception, but I think Nuitka said what module it happened in, which is not (yet) included, adding that now.

**Comment by xiezhipeng-git** [comment]
I don't know。I try Nuitka gym-tetris(pip install gym-tetris)  in windows.but always fail .It always say no *.so.And when add the *.so with WSL.It same error.And in [nuitka-crash-report.xml.txt](https://github.com/user-attachments/files/15949852/nuitka-crash-report.xml.txt) it is say gym.

**Comment by kayhayen** [comment]
Ok, that's great, I need a reproducer. I don't know pyinstaller unfortunately.

**Comment by kayhayen** [comment]
Thanks for your report, this is fixed on the factory branch, which is a development version under rapid development. You can try it out by going here: https://nuitka.net/doc/factory.html 

Feedback on whether this is working is very welcome. Please do not share plans to do it; only confirm or deny that it is working.

**Comment by kayhayen** [comment]
This is part of the current pre-release on the develop branch and will be in 2.4 release.

**Comment by kayhayen** [comment]
This is part of the stable release 2.4 that I just made.
